src/test/scala/com/github/tarao/slickjdbc/query/TranslatorSpec.scala

```diff
@@ -9,7 +9,7 @@ class ContextSpec extends UnitSpec {
     it("should return a caller") {
       val context = new Context {}
       context.caller.toString should fullyMatch regex (
-        "com[.]github[.]tarao[.]slickjdbc[.]query[.]ContextSpec[$].*[(]TranslatorSpec[.]scala:[0-9]+[)]"
+        "com[.]github[.]tarao[.]slickjdbc[.]query[.]ContextSpec.*[(]TranslatorSpec[.]scala:[0-9]+[)]"
```

Comment:
Scala-2.11 and Scala-2.12 generate different stack traces.

For example,

- In Scala-2.11.11: `com.github.tarao.slickjdbc.query.ContextSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(TranslatorSpec.scala:11)`
- In Scala-2.12.3: `com.github.tarao.slickjdbc.query.ContextSpec.$anonfun$new$2(TranslatorSpec.scala:11)`

I think this is because of scala-2.12 change

> Traits (#5003) and functions are compiled to their Java 8 equivalents. The compiler no longer generates trait implementation classes (T$class.class) and anonymous function classes (C$$anonfun$1.class).

http://www.scala-lang.org/news/2.12.0/#scala-212-overview